From df879920e9d2cc258ebc91e3497ba78a48f81786 Mon Sep 17 00:00:00 2001
From: unknown author <cooker@mandrivalinux.org>
Date: Mon, 5 Jan 2009 13:29:57 +0000
Subject: [PATCH 31/35] serial tag

---
 build/parsePreamble.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/build/parsePreamble.c b/build/parsePreamble.c
index 1ba7a8e..ae6127d 100644
--- a/build/parsePreamble.c
+++ b/build/parsePreamble.c
@@ -702,6 +702,7 @@ static struct PreambleRec_s preambleList[] = {
     {RPMTAG_VERSION,		0, 0, 0, "version"},
     {RPMTAG_RELEASE,		0, 0, 0, "release"},
     {RPMTAG_EPOCH,		0, 0, 0, "epoch"},
+    {RPMTAG_EPOCH,		0, 0, 0, "serial"},
     {RPMTAG_SUMMARY,		0, 1, 0, "summary"},
     {RPMTAG_LICENSE,		0, 0, 0, "license"},
     {RPMTAG_DISTRIBUTION,	0, 0, 0, "distribution"},
-- 
1.6.1.3

